charts/gardener/controlplane/charts/utils-common/templates/_secret-default-domain.yaml

```diff
@@ -11,12 +11,10 @@ metadata:
     chart: "{{ $.Chart.Name }}-{{ $.Chart.Version }}"
     release: "{{ $.Release.Name }}"
     heritage: "{{ $.Release.Service }}"
+    # TODO: remove in a future release
     garden.sapcloud.io/role: default-domain
```

Comment:
Why you remove the comment but not the deprecated `garden.sapcloud.io/role` label?

Reply:
To allow smooth update. If we remove the deprecated label in the same release that switches gardener-controller-manager to use the label from the new api group then there might be a small corner case in which the secret is first updated to contain only the label from the new api group and in the meantime gardener-controller-manager fails to resolve the default domain secret until it is updated with new version.

Reply:
OK, but can you keep the TOOD comment then?

Reply:
Which TODO comment?